app-i18n/ibus-theme-tools/Manifest

```diff
@@ -0,0 +1,2 @@
+DIST ibus-theme-tools-4.2.0.gh.tar.gz 38405 BLAKE2B 7fd5c15e8e4dea7a2c8a8b9456ced8e8a9600594feb2793af615daf448e6d05bf00287e5e56f85754962e324cfcb197dcfd87ee9b8201e925e919f21c2650ffa SHA512 5222324b2696140cb06b67a690967bd5320a411913c3fda11ef0198c9531b8938ad1a56f714bb7c42d90bba767aadbcbc7fedab7569a4c93f8f7471566298025
+DIST ibus-theme-tools-4.2.0.tar.gz 29477 BLAKE2B b170ec7933b802c7fe3ed949aa6f921a31c8670c580cf1060795bd5e42e1efa1124e371ed334623dbc5b30ccac2e0f2f1f9331713c90bf49913099844dfa72e5 SHA512 f2ba54edb0c814da97407f6b23d7e6cda8b389b0953663a1af3a66810cb6635640a20bdf3a50179aa5a6d79ebaeefcfdd4d5cf57fe9e24fdb59c968ba67d9193
```

Comment:
Hmm, it's downloading both of these. Are both required?

Reply:
No. Sorry about that. Just removed the GitHub source.